modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/services/fetcher/index.ts

```diff
@@ -62,9 +62,7 @@ const fetcher = async <T = any>(
 	});
 
 	if (!response.ok) {
-		const error = new FetcherError(
-			'An error occurred while fetching the data.'
-		);
+		const error = new FetcherError('This record does not exist');
```

Comment:
We cannot override this error message, but we can improve.

First we need to analyze the response type we are expecting to create an special type to address when resource does not exist

Then we can analyze the error and check during the error validation